pyproject2setuppy/entrypoint.py

```diff
@@ -0,0 +1,17 @@
+# external
+import attr
```

Comment:
Why do you need all that magic? It certainly doesn't make the code readable, and I'm pretty sure normal setuptools project don't need any fancy magic to declare entry points.